### PR TITLE
Update csrankings.csv

### PR DESCRIPTION
**Pull request by maudecouture**
Updated professors from Mila

By Maude Couture, the assistant from Jackie Cheung.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -306,6 +306,7 @@ Ailamaki Natassa,EPFL,https://people.epfl.ch/anastasia.ailamaki,YdrkeNwAAAAJ
 Aiman Erbad,Qatar University,http://www.qu.edu.qa/engineering/computer/faculty/aiman.php,6Pmbv8IAAAAJ
 Aiman Hanna,Concordia University,http://aimanhanna.com/concordia,NOSCHOLARPAGE
 Aimin Hao,Beihang University,http://scse.buaa.edu.cn/info/1078/2654.htm,NOSCHOLARPAGE
+Aishwarya Agrawal,Georgia Institute of Technology,https://www.cc.gatech.edu/grads/a/aagrawal307,pLurJcoAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:309]: Please correct the affiliation to McGill University.

Comment (emeryberger) [reply csrankings.csv:309]: (or Montreal, whichever it is)

File: csrankings.csv [hunk 2]
```diff
@@ -9024,6 +9030,7 @@ Joel P. Ilao,De La Salle University,https://www.dlsu.edu.ph/colleges/ccs/faculty
 Joel Reardon,University of Calgary,http://pages.cpsc.ucalgary.ca/~joel.reardon,NOSCHOLARPAGE
 Joel Spencer,New York University,http://cs.nyu.edu/spencer,AmmWYgEAAAAJ
 Joel Voldman,Massachusetts Institute of Technology,http://www.rle.mit.edu/biomicro,NOSCHOLARPAGE
+Joelle Pineau,McGill University,https://www.cs.mcgill.ca/~jpineau,CEt6_mMAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:9033]: Appointment must be >= 75% over the academic year to qualify for inclusion.

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by emeryberger** [comment]
bump @maudecouture

File: csrankings.csv (hunk 1 above)
Comment (maudecouture) [reply csrankings.csv:309]: Hello Emery , for some reason , Im not able to repush the pull request, I'm new at this...!

The correct affiliation for Aishwarya Agrawal it's University of Montréal

File: csrankings.csv (hunk 2 above)
Comment (maudecouture) [reply csrankings.csv:9033]: Not sure what it means, but Ok I will let know Joelle Pineau.

**Comment by maudecouture** [comment]
> bump @maudecouture

Sorry Im not familiair with Github, I never used it before.

**Comment by emeryberger** [comment]
Manually added faculty who do not appear to have > 25% industrial appointments. Omitted HEC, which is not yet in the database.

**Pull request closed without merging**